protocol/messenger_sync_profile_picture_test.go

```diff
@@ -27,10 +27,10 @@ func (s *MessengerSyncProfilePictureSuite) TestSyncProfilePicture() {
 	keyUID := s.m.account.KeyUID
 
 	// pair
-	theirMessenger, err := newMessengerWithKey(s.shh, s.privateKey, s.logger, nil)
-	s.Require().NoError(err)
+	theirMessenger := s.anotherMessenger()
+	defer TearDownMessenger(&s.Suite, theirMessenger)
```

Comment:
I haven't checked the Suite logic, and I don't know what goes wrong if we forget about adding this line. Perhaps the suite should keep track of all messenger instances and tear all of them down in the Teardown function?

Reply:
This pattern is repeated also for standalone messengers, i.e. not created by the Suite. In general, I like the idea, but I would leave it out of scope for this PR though.